roles/podman/tasks/main.yml

```
@@ -3,6 +3,11 @@
   package:
     name: 'podman'
```

Does it make sense to make this a list
```suggestion
    name:
      - 'podman'
      - 'shadow-utils'
```

---

I like keeping them separate.